src/test/resources/requests/allocated-listing-team-missing-request.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Use a single (templated) version of these request json files and dynamically modify the values as your stubs, this way we have less code to maintain.